migrations/2019-01-26-090348_create_version_owner_actions/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE version_owner_actions (
+    id SERIAL PRIMARY KEY,
+    version_id INTEGER REFERENCES versions(id) ON DELETE CASCADE NOT NULL,
+    owner_id INTEGER REFERENCES users(id) NOT NULL,
+    owner_token_id INTEGER REFERENCES api_tokens(id),
```

Comment:
What do you think about calling this `api_token_id` instead?

Reply:
I agree with you but this is no longer in this PR. It's in a PR that's already been merged.

Would you like me to rename the field?

Reply:
Same with this spot, up to you whether you'd like to add a renaming migration to this PR.